src/main/java/edu/wisc/my/ltiproxy/dao/LTILaunchPropertyFileDao.java

```diff
@@ -5,12 +5,13 @@
 
 import com.fasterxml.jackson.core.JsonParseException;
 import com.fasterxml.jackson.databind.JsonMappingException;
+import com.google.common.collect.Multimap;
 
 public interface LTILaunchPropertyFileDao{
     
     public Map<String, String> getLaunchParameters(String key) throws JsonParseException, JsonMappingException, IOException;
     
-    public Map<String, String[]> getHeadersToReplace(String key) throws JsonParseException, JsonMappingException, IOException;
+    public Multimap<String, String> getHeadersToReplace(String key) throws JsonParseException, JsonMappingException, IOException;
```

Comment:
It'd be interesting to articulate, for Semantic Versioning purposes, what the "API" of this software product is. Is it the Java API? Is it just the effective exposed REST web service? If it's the Java API, then this starts to look like a breaking change that yields a `2.0.0` release. Which isn't necessarily a bad thing at all, this project is just getting started. No known in-the-wild adopters to inconvenience, as it were.